tools/pom.xml

```diff
@@ -45,6 +50,11 @@
 			<artifactId>bcprov-jdk18on</artifactId>
 			<version>1.76</version>
 		</dependency>
+		<dependency>
+			<groupId>org.bouncycastle</groupId>
+			<artifactId>bcpkix-jdk18on</artifactId>
```

Comment:
does the jdk18on stand for java 18? If yes, why that and not for java 17(if that exists)?

Reply:
This jar contains APIs for JDK 1.8 and up.
[bcpkix-jdk15on](https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on) contains APIs for JDK 1.5 and up. 
Guess there are no java11, 17, 18, 21+ dedicated dependencies

We might try to upgrade to latest <version>1.80</version>, since this 1.76 is from July 2023